flixel/text/FlxBitmapText.hx

```diff
@@ -1461,7 +1462,7 @@ class FlxBitmapText extends FlxSprite
 	 * Set border's style (shadow, outline, etc), color, and size all in one go!
 	 * 
 	 * @param	Style outline style
-	 * @param	Color outline color in flash 0xRRGGBB format
+	 * @param	Color outline color in flash 0xAARRGGBB format
```

Comment:
The method description here was copied from `FlxText`, where the border color does indeed ignore alpha values. Not a big fan of this behavior, the two text classes should be as consistent as possible I think... I wonder if transparency is possible in FlxText's borderColor.